src/Simulation/Simulators/OpenSystemsSimulator/DataModel/State.cs

```diff
@@ -1,116 +1,107 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-#nullable enable
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Numerics;
-using System.Text.Json;
 using System.Text.Json.Serialization;
 using NumSharp;
 using static System.Math;
 
-namespace Microsoft.Quantum.Experimental
+namespace Microsoft.Quantum.Experimental;
+
+[JsonConverter(typeof(StateConverter))]
+public abstract class State
 {
-    [JsonConverter(typeof(StateConverter))]
-    public abstract class State
-    {
-        [JsonPropertyName("n_qubits")]
-        public int NQubits { get; set; } = 1;
+    [JsonPropertyName("n_qubits")]
+    public int NQubits { get; set; } = 1;
 
-        internal State()
-        { }
+    internal State()
+    { }
 
-        internal State(int nQubits)
-        {
-            NQubits = nQubits;
-        }
+    internal State(int nQubits)
+    {
+        NQubits = nQubits;
     }
+}
 
-    public class StabilizerState : State
+public class StabilizerState : State
+{
+    // Design note:
+    // We could use the same array converter as used of complex-like arrays
```

Comment:
I have an impression of some typo in the highlighted fragment
> // We could use the same array converter **as used of complex-like arrays**

Reply:
```suggestion
    // We could use the same array converter as used to convert complex-like arrays
```

Reply:
Looks more clear, thanks.